src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2161,7 +2161,6 @@ bool AdaptDispatch::SetCursorStyle(const DispatchTypes::CursorStyle cursorStyle)
     switch (cursorStyle)
     {
     case DispatchTypes::CursorStyle::BlinkingBlock:
```

Comment:
if i recall conhost correctly, this won't do what we're intending. This will map `0` to legacy (?) even though there's an experimental setting (property sheet, "terminal" page) to set the default cursor style.

Reply:
Does `experimental` means that it's only available after certain windows build?

Reply:
It's available in 18362 and higher. https://devblogs.microsoft.com/commandline/new-experimental-console-features/

Reply:
I found the settings struct called `CONSOLE_STATE_INFO` but it's not exposed to `congetset`, isn't it.

See this is what happens when you found a "Starter" issue turns out to be a "Pro Plus" issue.

I'm kidding. Give me those "Enterprise" issues next time.